app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Admin::UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    redirect_to admin_users_path, :notice => "User has been deleted"
```

Comment:
Use a new syntax. i.e. `notice: 'msg'`
What is @destroy fails? It will still inform about the successful deletion. Please address this.